src/BoxWhisker.js

```diff
@@ -24,6 +24,22 @@ export default class BoxWhisker extends Plot {
     this._discrete = "x";
     this._shape = constant("Box");
     this.x("x");
+    this._shapeConfig = assign(this._shapeConfig, {
+      Box: {
+        label: (d, i) => this._drawLabel(d, i)
+      }
+    });
+
+    this._drawLabel = (d, i) => {
```

Comment:
We cannot override the default for `this._drawLabel`, as this behavior is only going to be used for Boxes, and someone may want to create a chart combining Boxes with other shapes using `Plot` directly. But this function directly inline inside of the shapeConfig for Box. There is no need to save it to a variable as it is only used in 1 place.